### PR TITLE
Update Amazon Linux images (hello, arm64v8)

### PR DESCRIPTION
**Pull request by ilianaw**
Hopefully this is correctly done... I was able to run `bashbrew cat` and get correct output out but couldn't convince bashbrew to clone from a file URL to test :)

### CODE REVIEW

**Comment by tianon** [comment]
Probably want to check out the discussion on https://github.com/docker-library/official-images/pull/5073#issuecomment-440460027, which is pretty similar to what's missing here (if that's not enough to help explain, feel free to poke again and I'll work on a more concrete example for Amazon Linux). :+1: :heart:

**Comment by ilianaw** [comment]
I had some local changes that weren't in the commit :upside_down_face:

File: library/amazonlinux [hunk 1]
```diff
@@ -4,13 +4,21 @@ Maintainers: Amazon Linux Team <amazon-linux@amazon.com> (@aws),
              Eric Warehime (@Eric-Warehime)
 GitRepo: https://github.com/aws/amazon-linux-docker-images.git
 
-Tags: 2.0.20181010, 2, latest
-GitFetch: refs/heads/amzn2
-GitCommit: 1a13f4b999a22b95b0d8bbd10d4190e9fc1b43fd
+Tags: 2.0.20181114, 2, latest
+Architectures: amd64, arm64v8
+GitCommit: 3de85bea51cc97f74263a12825564db727b55996
```
Comment (tianon) [inline library/amazonlinux]: So, this is fine, but could potentially lead to `amd64` bits being pushed to a different architecture (typo in `ARCH-GitCommit:` field name, for example), so I'd recommend pointing this global value to some useless commit from `master` that'll explicitly fail to build (and then you can move it up to the global section as well so it doesn't have to be duplicated).

Comment (ilianaw) [reply library/amazonlinux]: OK, I'll just use the latest commit from master, which is missing a Dockerfile.

Comment (ilianaw) [reply library/amazonlinux]: I missed your text about moving it up to the global section... I'll do that next time :)

**Comment by tianon** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 5380a0c..0959f18 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,4 +1,6 @@
 amazonlinux:1 @ amd64
 amazonlinux:1-with-sources @ amd64
 amazonlinux:latest @ amd64
+amazonlinux:latest @ arm64v8
 amazonlinux:with-sources @ amd64
+amazonlinux:with-sources @ arm64v8
diff --git a/_bashbrew-list b/_bashbrew-list
index 7759a38..0e41428 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -2,8 +2,8 @@ amazonlinux:1
 amazonlinux:1-with-sources
 amazonlinux:2
 amazonlinux:2-with-sources
-amazonlinux:2.0.20181010
-amazonlinux:2.0.20181010-with-sources
+amazonlinux:2.0.20181114
+amazonlinux:2.0.20181114-with-sources
 amazonlinux:2018.03
 amazonlinux:2018.03-with-sources
 amazonlinux:2018.03.0.20180827
diff --git a/amazonlinux_latest/Dockerfile b/amazonlinux_latest/Dockerfile
index d5d296e..32d10db 100644
--- a/amazonlinux_latest/Dockerfile
+++ b/amazonlinux_latest/Dockerfile
@@ -1,3 +1,3 @@
 FROM scratch
-ADD amzn2-container-raw-2.0.20181010-x86_64.tar.xz /
+ADD amzn2-container-raw-2.0.20181114-arm64.tar.xz /
 CMD ["/bin/bash"]
diff --git a/amazonlinux_latest/amzn2-container-raw-2.0.20181010-x86_64.tar.xz b/amazonlinux_latest/amzn2-container-raw-2.0.20181114-arm64.tar.xz
similarity index 73%
rename from amazonlinux_latest/amzn2-container-raw-2.0.20181010-x86_64.tar.xz
rename to amazonlinux_latest/amzn2-container-raw-2.0.20181114-arm64.tar.xz
index b9934dc..f0357fd 100644
Binary files a/amazonlinux_latest/amzn2-container-raw-2.0.20181010-x86_64.tar.xz and b/amazonlinux_latest/amzn2-container-raw-2.0.20181114-arm64.tar.xz differ
diff --git a/amazonlinux_latest/amzn2-container-raw-2.0.20181010-x86_64.tar.xz  'tar -t' b/amazonlinux_latest/amzn2-container-raw-2.0.20181114-arm64.tar.xz  'tar -t'
similarity index 99%
rename from amazonlinux_latest/amzn2-container-raw-2.0.20181010-x86_64.tar.xz  'tar -t'
rename to amazonlinux_latest/amzn2-container-raw-2.0.20181114-arm64.tar.xz  'tar -t'
index a09c545..a87deaa 100644
--- a/amazonlinux_latest/amzn2-container-raw-2.0.20181010-x86_64.tar.xz  'tar -t'	
+++ b/amazonlinux_latest/amzn2-container-raw-2.0.20181114-arm64.tar.xz  'tar -t'	
@@ -16,7 +16,7 @@ etc/
 etc/adjtime
 etc/aliases
 etc/alternatives/
-etc/alternatives/libnssckbi.so.x86_64
+etc/alternatives/libnssckbi.so
 etc/bash_completion.d/
 etc/bashrc
 etc/chkconfig.d/
@@ -64,6 +64,10 @@ etc/mtab
 etc/nsswitch.conf
 etc/openldap/
 etc/openldap/certs/
+etc/openldap/certs/cert8.db
+etc/openldap/certs/key3.db
+etc/openldap/certs/password
+etc/openldap/certs/secmod.db
 etc/openldap/ldap.conf
 etc/opt/
 etc/os-release
@@ -735,7 +739,7 @@ usr/lib64/krb5/plugins/preauth/
 usr/lib64/krb5/plugins/tls/
 usr/lib64/krb5/plugins/tls/k5tls.so
 usr/lib64/ld-2.26.so
-usr/lib64/ld-linux-x86-64.so.2
+usr/lib64/ld-linux-aarch64.so.1
 usr/lib64/libacl.so.1
 usr/lib64/libacl.so.1.1.0
 usr/lib64/libanl-2.26.so
@@ -855,8 +859,6 @@ usr/lib64/libmetalink.so.3.0.0
 usr/lib64/libmount.so.1
 usr/lib64/libmount.so.1.1.0
 usr/lib64/libm.so.6
-usr/lib64/libmvec-2.26.so
-usr/lib64/libmvec.so.1
 usr/lib64/libncurses.so.6
 usr/lib64/libncurses.so.6.0
 usr/lib64/libncursesw.so.6
@@ -3309,6 +3311,7 @@ usr/libexec/p11-kit/p11-kit-remote
 usr/libexec/p11-kit/trust-extract-compat
 usr/libexec/urlgrabber-ext-down
 usr/lib/games/
+usr/lib/ld-linux-aarch64.so.1
 usr/lib/locale/
 usr/lib/locale/C.utf8/
 usr/lib/locale/C.utf8/LC_ADDRESS
@@ -3804,11 +3807,11 @@ usr/lib/modules/
 usr/lib/python2.7/
 usr/lib/python2.7/site-packages/
 usr/lib/python2.7/site-packages/amazon_linux_extras/
-usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.3-py2.7.egg-info/
-usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.3-py2.7.egg-info/dependency_links.txt
-usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.3-py2.7.egg-info/PKG-INFO
-usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.3-py2.7.egg-info/SOURCES.txt
-usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.3-py2.7.egg-info/top_level.txt
+usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.4-py2.7.egg-info/
+usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.4-py2.7.egg-info/dependency_links.txt
+usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.4-py2.7.egg-info/PKG-INFO
+usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.4-py2.7.egg-info/SOURCES.txt
+usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.4-py2.7.egg-info/top_level.txt
 usr/lib/python2.7/site-packages/amazon_linux_extras/cli.py
 usr/lib/python2.7/site-packages/amazon_linux_extras/cli.pyc
 usr/lib/python2.7/site-packages/amazon_linux_extras/cli.pyo
@@ -4122,9 +4125,9 @@ usr/sbin/chkconfig
 usr/sbin/chroot
 usr/sbin/getcap
 usr/sbin/getpcaps
-usr/sbin/glibc_post_upgrade.x86_64
+usr/sbin/glibc_post_upgrade.aarch64
 usr/sbin/iconvconfig
-usr/sbin/iconvconfig.x86_64
+usr/sbin/iconvconfig.aarch64
 usr/sbin/install-info
 usr/sbin/ldconfig
 usr/sbin/sasldblistusers2
@@ -4175,8 +4178,8 @@ usr/share/bash-completion/completions/yummain.py
 usr/share/desktop-directories/
 usr/share/dict/
 usr/share/doc/
-usr/share/doc/amazon-linux-extras-1.6.3/
-usr/share/doc/amazon-linux-extras-1.6.3/README
+usr/share/doc/amazon-linux-extras-1.6.4/
+usr/share/doc/amazon-linux-extras-1.6.4/README
 usr/share/doc/bash-4.2.46/
 usr/share/doc/bash-4.2.46/COPYING
 usr/share/doc/bzip2-libs-1.0.6/
@@ -9832,11 +9835,11 @@ var/cache/
 var/cache/ldconfig/
 var/cache/ldconfig/aux-cache
 var/cache/yum/
-var/cache/yum/x86_64/
-var/cache/yum/x86_64/2/
-var/cache/yum/x86_64/2/amzn2-core/
-var/cache/yum/x86_64/2/amzn2-core/gen/
-var/cache/yum/x86_64/2/amzn2-core/packages/
+var/cache/yum/aarch64/
+var/cache/yum/aarch64/2/
+var/cache/yum/aarch64/2/amzn2-core/
+var/cache/yum/aarch64/2/amzn2-core/gen/
+var/cache/yum/aarch64/2/amzn2-core/packages/
 var/db/
 var/empty/
 var/games/
@@ -9846,7 +9849,7 @@ var/kerberos/krb5/
 var/kerberos/krb5/user/
 var/lib/
 var/lib/alternatives/
-var/lib/alternatives/libnssckbi.so.x86_64
+var/lib/alternatives/libnssckbi.so
 var/lib/games/
 var/lib/misc/
 var/lib/rpm/
@@ -9870,6 +9873,9 @@ var/lib/rpm/Sigmd5
 var/lib/rpm-state/
 var/lib/rpm/Triggername
 var/lib/yum/
+var/lib/yum/groups/
+var/lib/yum/groups/environment
+var/lib/yum/groups/installed
 var/lib/yum/uuid
 var/local/
 var/lock
diff --git a/amazonlinux_with-sources/Dockerfile b/amazonlinux_with-sources/Dockerfile
index 26f58f7..a6d8947 100644
--- a/amazonlinux_with-sources/Dockerfile
+++ b/amazonlinux_with-sources/Dockerfile
@@ -1,6 +1,6 @@
 FROM scratch
-ADD amzn2-container-raw-2.0.20181010-x86_64.tar.xz /
+ADD amzn2-container-raw-2.0.20181114-arm64.tar.xz /
 CMD ["/bin/bash"]
 RUN mkdir /usr/src/srpm \
- && curl -o /usr/src/srpm/srpm-bundle.tar.gz "https://amazon-linux-docker-sources.s3-accelerate.amazonaws.com/amzn2/srpm-bundle.tar.gz?versionId=_uFmafMKERILPq3MZ7xKNOyUR6wN_m9d" \
- && echo "8adebd309defad308f0d3216a73fdf90171657df951abc07630e6f4083d6d718 /usr/src/srpm/srpm-bundle.tar.gz" | sha256sum -c -
+ && curl -o /usr/src/srpm/srpm-bundle.tar.gz "https://amazon-linux-docker-sources.s3-accelerate.amazonaws.com/amzn2/srpm-bundle.20181114.tar.gz?versionId=nB5.4dIHFJSZqiKkbJWmAufA2zT96NtU" \
+ && echo "9e69458ec7d3617a02584b64d77558c513253de6ef0d33b8efbd709430c6e14a /usr/src/srpm/srpm-bundle.tar.gz" | sha256sum -c -
diff --git a/amazonlinux_with-sources/amzn2-container-raw-2.0.20181010-x86_64.tar.xz b/amazonlinux_with-sources/amzn2-container-raw-2.0.20181114-arm64.tar.xz
similarity index 73%
rename from amazonlinux_with-sources/amzn2-container-raw-2.0.20181010-x86_64.tar.xz
rename to amazonlinux_with-sources/amzn2-container-raw-2.0.20181114-arm64.tar.xz
index b9934dc..f0357fd 100644
Binary files a/amazonlinux_with-sources/amzn2-container-raw-2.0.20181010-x86_64.tar.xz and b/amazonlinux_with-sources/amzn2-container-raw-2.0.20181114-arm64.tar.xz differ
diff --git a/amazonlinux_with-sources/amzn2-container-raw-2.0.20181010-x86_64.tar.xz  'tar -t' b/amazonlinux_with-sources/amzn2-container-raw-2.0.20181114-arm64.tar.xz  'tar -t'
similarity index 99%
rename from amazonlinux_with-sources/amzn2-container-raw-2.0.20181010-x86_64.tar.xz  'tar -t'
rename to amazonlinux_with-sources/amzn2-container-raw-2.0.20181114-arm64.tar.xz  'tar -t'
index a09c545..a87deaa 100644
--- a/amazonlinux_with-sources/amzn2-container-raw-2.0.20181010-x86_64.tar.xz  'tar -t'	
+++ b/amazonlinux_with-sources/amzn2-container-raw-2.0.20181114-arm64.tar.xz  'tar -t'	
@@ -16,7 +16,7 @@ etc/
 etc/adjtime
 etc/aliases
 etc/alternatives/
-etc/alternatives/libnssckbi.so.x86_64
+etc/alternatives/libnssckbi.so
 etc/bash_completion.d/
 etc/bashrc
 etc/chkconfig.d/
@@ -64,6 +64,10 @@ etc/mtab
 etc/nsswitch.conf
 etc/openldap/
 etc/openldap/certs/
+etc/openldap/certs/cert8.db
+etc/openldap/certs/key3.db
+etc/openldap/certs/password
+etc/openldap/certs/secmod.db
 etc/openldap/ldap.conf
 etc/opt/
 etc/os-release
@@ -735,7 +739,7 @@ usr/lib64/krb5/plugins/preauth/
 usr/lib64/krb5/plugins/tls/
 usr/lib64/krb5/plugins/tls/k5tls.so
 usr/lib64/ld-2.26.so
-usr/lib64/ld-linux-x86-64.so.2
+usr/lib64/ld-linux-aarch64.so.1
 usr/lib64/libacl.so.1
 usr/lib64/libacl.so.1.1.0
 usr/lib64/libanl-2.26.so
@@ -855,8 +859,6 @@ usr/lib64/libmetalink.so.3.0.0
 usr/lib64/libmount.so.1
 usr/lib64/libmount.so.1.1.0
 usr/lib64/libm.so.6
-usr/lib64/libmvec-2.26.so
-usr/lib64/libmvec.so.1
 usr/lib64/libncurses.so.6
 usr/lib64/libncurses.so.6.0
 usr/lib64/libncursesw.so.6
@@ -3309,6 +3311,7 @@ usr/libexec/p11-kit/p11-kit-remote
 usr/libexec/p11-kit/trust-extract-compat
 usr/libexec/urlgrabber-ext-down
 usr/lib/games/
+usr/lib/ld-linux-aarch64.so.1
 usr/lib/locale/
 usr/lib/locale/C.utf8/
 usr/lib/locale/C.utf8/LC_ADDRESS
@@ -3804,11 +3807,11 @@ usr/lib/modules/
 usr/lib/python2.7/
 usr/lib/python2.7/site-packages/
 usr/lib/python2.7/site-packages/amazon_linux_extras/
-usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.3-py2.7.egg-info/
-usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.3-py2.7.egg-info/dependency_links.txt
-usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.3-py2.7.egg-info/PKG-INFO
-usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.3-py2.7.egg-info/SOURCES.txt
-usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.3-py2.7.egg-info/top_level.txt
+usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.4-py2.7.egg-info/
+usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.4-py2.7.egg-info/dependency_links.txt
+usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.4-py2.7.egg-info/PKG-INFO
+usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.4-py2.7.egg-info/SOURCES.txt
+usr/lib/python2.7/site-packages/amazon_linux_extras-1.6.4-py2.7.egg-info/top_level.txt
 usr/lib/python2.7/site-packages/amazon_linux_extras/cli.py
 usr/lib/python2.7/site-packages/amazon_linux_extras/cli.pyc
 usr/lib/python2.7/site-packages/amazon_linux_extras/cli.pyo
@@ -4122,9 +4125,9 @@ usr/sbin/chkconfig
 usr/sbin/chroot
 usr/sbin/getcap
 usr/sbin/getpcaps
-usr/sbin/glibc_post_upgrade.x86_64
+usr/sbin/glibc_post_upgrade.aarch64
 usr/sbin/iconvconfig
-usr/sbin/iconvconfig.x86_64
+usr/sbin/iconvconfig.aarch64
 usr/sbin/install-info
 usr/sbin/ldconfig
 usr/sbin/sasldblistusers2
@@ -4175,8 +4178,8 @@ usr/share/bash-completion/completions/yummain.py
 usr/share/desktop-directories/
 usr/share/dict/
 usr/share/doc/
-usr/share/doc/amazon-linux-extras-1.6.3/
-usr/share/doc/amazon-linux-extras-1.6.3/README
+usr/share/doc/amazon-linux-extras-1.6.4/
+usr/share/doc/amazon-linux-extras-1.6.4/README
 usr/share/doc/bash-4.2.46/
 usr/share/doc/bash-4.2.46/COPYING
 usr/share/doc/bzip2-libs-1.0.6/
@@ -9832,11 +9835,11 @@ var/cache/
 var/cache/ldconfig/
 var/cache/ldconfig/aux-cache
 var/cache/yum/
-var/cache/yum/x86_64/
-var/cache/yum/x86_64/2/
-var/cache/yum/x86_64/2/amzn2-core/
-var/cache/yum/x86_64/2/amzn2-core/gen/
-var/cache/yum/x86_64/2/amzn2-core/packages/
+var/cache/yum/aarch64/
+var/cache/yum/aarch64/2/
+var/cache/yum/aarch64/2/amzn2-core/
+var/cache/yum/aarch64/2/amzn2-core/gen/
+var/cache/yum/aarch64/2/amzn2-core/packages/
 var/db/
 var/empty/
 var/games/
@@ -9846,7 +9849,7 @@ var/kerberos/krb5/
 var/kerberos/krb5/user/
 var/lib/
 var/lib/alternatives/
-var/lib/alternatives/libnssckbi.so.x86_64
+var/lib/alternatives/libnssckbi.so
 var/lib/games/
 var/lib/misc/
 var/lib/rpm/
@@ -9870,6 +9873,9 @@ var/lib/rpm/Sigmd5
 var/lib/rpm-state/
 var/lib/rpm/Triggername
 var/lib/yum/
+var/lib/yum/groups/
+var/lib/yum/groups/environment
+var/lib/yum/groups/installed
 var/lib/yum/uuid
 var/local/
 var/lock
````

</details>

**Comment by tianon** [comment]
LGTM

Build test of #5108; dd73c056; `arm64v8` (`amazonlinux`):

```console

$ bashbrew build amazonlinux:2.0.20181114
Building bashbrew/cache:13bb1ba7f4504e4437c8a9259d0437cae2944d85da1d01432bb7d632bf72da0b (amazonlinux:2.0.20181114)
Tagging amazonlinux:2.0.20181114
Tagging amazonlinux:2
Tagging amazonlinux:latest

$ test/run.sh amazonlinux:2.0.20181114
testing amazonlinux:2.0.20181114
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': '*LOCK*'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build amazonlinux:2.0.20181114-with-sources
Building bashbrew/cache:aeca8b24d5db5ed6878c2e2f9b523a15e43e3333ec7fd67fedc2382f7602260c (amazonlinux:2.0.20181114-with-sources)
Tagging amazonlinux:2.0.20181114-with-sources
Tagging amazonlinux:2-with-sources
Tagging amazonlinux:with-sources

$ test/run.sh amazonlinux:2.0.20181114-with-sources
testing amazonlinux:2.0.20181114-with-sources
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': '*LOCK*'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build amazonlinux:2018.03.0.20180827
skipping "amazonlinux:2018.03.0.20180827" (due to architecture "arm64v8"; only "amd64" supported)

$ test/run.sh amazonlinux:2018.03.0.20180827
testing amazonlinux:2018.03.0.20180827
	image does not exist!


$ bashbrew build amazonlinux:2018.03.0.20180827-with-sources
skipping "amazonlinux:2018.03.0.20180827-with-sources" (due to architecture "arm64v8"; only "amd64" supported)

$ test/run.sh amazonlinux:2018.03.0.20180827-with-sources
testing amazonlinux:2018.03.0.20180827-with-sources
	image does not exist!

```


The following images failed at least one test:

- `amazonlinux`:
  - `amazonlinux:2018.03.0.20180827`
  - `amazonlinux:2018.03.0.20180827-with-sources`

Build test of #5108; dd73c056; `amd64` (`amazonlinux`):

```console

$ bashbrew build amazonlinux:2.0.20181114
Building bashbrew/cache:b096c9f63db3f653cbc03934496e3d88e0da41c4cca1d95a5181fe457b30f391 (amazonlinux:2.0.20181114)
Tagging amazonlinux:2.0.20181114
Tagging amazonlinux:2
Tagging amazonlinux:latest

$ test/run.sh amazonlinux:2.0.20181114
testing amazonlinux:2.0.20181114
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': '*LOCK*'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build amazonlinux:2.0.20181114-with-sources
Building bashbrew/cache:451b4e6df88f950b9bd59ab916cfc88b0b1e32ee70e810a176645b2fd903a0a6 (amazonlinux:2.0.20181114-with-sources)
Tagging amazonlinux:2.0.20181114-with-sources
Tagging amazonlinux:2-with-sources
Tagging amazonlinux:with-sources

$ test/run.sh amazonlinux:2.0.20181114-with-sources
testing amazonlinux:2.0.20181114-with-sources
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': '*LOCK*'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build amazonlinux:2018.03.0.20180827
Building bashbrew/cache:c8a1514337c2a32a251df6a8625d218e978e7f73d84fde2cdc5367c89184f8c2 (amazonlinux:2018.03.0.20180827)
Tagging amazonlinux:2018.03.0.20180827
Tagging amazonlinux:2018.03
Tagging amazonlinux:1

$ test/run.sh amazonlinux:2018.03.0.20180827
testing amazonlinux:2018.03.0.20180827
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build amazonlinux:2018.03.0.20180827-with-sources
Building bashbrew/cache:fabe15a18bd0829adc9288f75bbf6041bc2b8d3e39b9bfa719e999db29f8aa30 (amazonlinux:2018.03.0.20180827-with-sources)
Tagging amazonlinux:2018.03.0.20180827-with-sources
Tagging amazonlinux:2018.03-with-sources
Tagging amazonlinux:1-with-sources

$ test/run.sh amazonlinux:2018.03.0.20180827-with-sources
testing amazonlinux:2018.03.0.20180827-with-sources
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**